NixOS Build fails with "patchelf: command not found"

Cloning the current master and running `make` in `nix run` produces
```
You can specify an upload port by editing makeConfig.mk.
Installing development environment...
Submodule 'arduino-esp32' (https://github.com/espressif/arduino-esp32) registered for path 'arduino-esp32'
Cloning into '/home/gaisseml/dev/LED-Beleuchtung/arduino-esp32'...
Submodule path 'arduino-esp32': checked out 'e346f20aa9f455f3147d9d018e09dd7c05818951'
System: Linux, Info: Linux-5.0.8-x86_64-with-glibc2.2.5
Platform: x86_64-pc-linux-gnu
Downloading xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar.gz
Done
Extracting xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar.gz
Downloading esptool-2.3.1-linux.tar.gz
Done
Extracting esptool-2.3.1-linux.tar.gz
Downloading mkspiffs-0.2.3-arduino-esp32-linux64.tar.gz
Done
Extracting mkspiffs-0.2.3-arduino-esp32-linux64.tar.gz
Renaming mkspiffs-0.2.3-arduino-esp32-linux64 to mkspiffs
Done
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 4: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 8: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 12: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 16: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 20: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 24: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 27: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 31: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 35: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 39: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 43: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 47: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 51: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 55: patchelf: command not found
/nix/store/la4l0zb2kd8gfnjbmznqjb3sriq845sa-patch-script/bin/patch-script: line 59: patchelf: command not found
make: *** [Makefile:117: arduino-esp32/tools/esptool/esptool.py] Error 127
```

--- a/default.nix
+++ b/default.nix
@@ -50,5 +50,6 @@ buildEnv {
     gnumake
     mkspiffs
     (python3.withPackages (pkgs: with pkgs; [ pyserial ]))
+    patchelf
   ];
 }

